src/index.html

```diff
@@ -1,13 +1,61 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="card_page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="styles/index.css">
+  </head>
+  <body class="card">
+    <div 
+      class="container"
+      data-qa="card"
+    >
+      <img 
+        src="images/imac.jpeg"
+        alt="Apple A1419 iMac 27 Retina 5K Monoblock "
+        class="container__img"
+      >
+      <p class="container__description">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </p>
```

Comment:
you forgot tag <h2>
it's a good place for it
![image](https://user-images.githubusercontent.com/87998640/221668242-caa741f0-05f1-4334-85f3-ca304274a8eb.png)

Reply:
agree, but up to you